tests/test_project/tests/test_password_history.py

```diff
@@ -100,3 +100,179 @@ def test_change_number_hasher_iterations(self):
                     user_config__iterations=HistoryVeryStrongHasher.iterations).count(),
                 1,
             )
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 5
+        }
+    }])
+    def test_lookup_range_true_positive(self):
+        # We're considering the last 5 passwords and setting a valid password
+        self.create_user(1)
```

Comment:
@spokerman12, since you are repeating the same function calls in consecutive lines with just the password number incremented, can you use a `for` loop to shorten the code?